app/services/project_queries/publish_service.rb

```diff
@@ -37,7 +37,7 @@ def after_validate(service_call)
     end
 
     def persist(service_call)
-      model.save
+      model.save(validate: false)
```

Comment:
This line deserves a short comment, I think. After all, validations are usually something we want. But not when sharing a project query. Why? Future readers should know. The same comment could also be added to `PublishContract#validate_model?`

There is a nice feature spec for this, but just from reading the business logic, it's hard to figure out why we do this.

Reply:
Comment added.